server/mesh.go

```diff
@@ -18,6 +18,12 @@ func NewMeshHandler(loggerFactory LoggerFactory, wss *WSS, active_rooms map[stri
 
 	fn := func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Just curios, I am not so experienced in Go, but is it ok to use one letter shortcuts? In JS it's really bad code smell